### PR TITLE
[DRWTSN32] Spanish Translation

### PR DESCRIPTION
**Pull request by elhoir**
Spanish translation was not created. Fix it.

### CODE REVIEW

**Comment by Swyter** [comment]
The other thing would be avoiding this kind of super small postfix commits. Git has the `amend` function, multiple commits make sense when you want to spkit your work by components or logical parts that make it easier to review or navigate the project's history.

**Comment by elhoir** [comment]
i´m still just learning git & github. Sorry. I`ll try next time.

**Comment by Swyter** [comment]
No worries, git is hard to use. The content is the important part. :)

**Review by HBelusca — APPROVED** [review]

File: base/applications/drwtsn32/lang/es-ES.rc [hunk 1]
```diff
@@ -0,0 +1,15 @@
+/*
+ * PROJECT:     Dr. Watson crash reporter
+ * LICENSE:     GPL-2.0+ (https://spdx.org/licenses/GPL-2.0+)
+ * PURPOSE:     es-ES resource file
+ * COPYRIGHT:   Copyright 2018 William Kent (wjk011 [at] gmail [dot] com)
+ * TRANSLATOR:  Javier Fernández, aka "Elhoir" 2021 <elhoir AT gmail DOT com>
```
Comment (HBelusca) [inline base/applications/drwtsn32/lang/es-ES.rc:6]: Note: Other language files here replace the Copyright (of the file) by the translator.

Comment (elhoir) [reply base/applications/drwtsn32/lang/es-ES.rc:6]: Ah, i supposed that line refers the software itself, not the locale file. Thanks for pointing.

Comment (HBelusca) [inline base/applications/drwtsn32/lang/es-ES.rc:6]:
```suggestion
 * COPYRIGHT:   Copyright 2021 Javier Fernández, aka "Elhoir" <elhoir AT gmail DOT com>
```

**Comment by HBelusca** [comment]
@Swyter And where are they? github doesn't show any other suggestions (as comments) here.

**Comment by Swyter** [comment]
The pending reviews.

**Comment by Swyter** [comment]
Seems like they only show up when I'm logged in, great.
IMG_20211230_012904

**Comment by binarymaster** [comment]
So you did not submitted these reviews? :eyes:

**Comment by Swyter** [comment]
Seems like I misinterpreted what pending meant, yeah. My bad.

**Comment by binarymaster** [comment]
Perhaps you can try to click "Finish review", but I don't remember if that's available when PR is already merged.

If it's not available, you can open a new PR with your suggestions.

**Comment by Swyter** [comment]
Boy, is this embarrassing. I'm usually good with these things. Sorry again.

**Comment by elhoir** [comment]
aarrrrggghh

**Pull request merged**